.circleci/readme.md

```diff
@@ -25,22 +25,27 @@ The file `config.yml` is the configuration file that is read by CircleCI. This f
 automatically generated by the `generate.sh` script from the `config-2_1.yml` file.
 
 The provided `config.yml` file uses low resources so users of the CircleCI free tier can
```

Comment:
nit... ish?

Maybe we could just phrase this as something like:
```
The provided default `config.yaml` file targets a subset of tests that succeed on the free tier of CircleCI; this is a duplicate of the config.yml.FREE file for generating purposes. There is also an optional config.yml.PAID file that includes more test suites that require the higher resources available on premium CircleCI accounts, which we notably also use for validating new releases.
```

Reply:
I think we can update this when we actually remove the tests that require paid resources from `config.yml.FREE`. I was thinking on doing that on a separate ticket, so we get rid of HIGHRES as soon and painlessly as possible, since that eases any further improvement. wdyt?

Reply:
👍